packages/maskbook/src/extension/options-page/DashboardRouters/Wallets.tsx

```diff
@@ -84,6 +85,11 @@ const useStyles = makeStyles((theme) =>
     }),
 )
 
+export const DashboardWalletsContext = createContext<{
+    detailedTokens: AssetDetailed[]
+    retryDetailedTokens: () => void
+}>(null!)
```

Comment:
Dose it necessary to give an empty implementation here?

Reply:
I think it's okay to use `null!` since it will be initialized here: https://github.com/DimensionDev/Maskbook/pull/2216/files#diff-ebc6683e3022ce8a049715586713e2d796eae531aa5b5e96692e98762a355d41R155